### PR TITLE
Chore: Switch from `sass` to `sass-embedded`

### PR DESCRIPTION
**Pull request by literat**
* @see: https://sass-lang.com/blog/embedded-sass-is-live/
* more performand
* modern API

### CODE REVIEW

**Review by OndraM — CHANGES_REQUESTED** [review]
I merged the previous PRs, could you please rebase?

File: package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
     "postcss-cli": "11.0.0",
     "prettier": "3.3.3",
     "replace-in-file": "8.2.0",
-    "sass": "1.80.1",
+    "sass-embedded": "^1.80.3",
```
Comment (OndraM) [inline package.json:112]: Also pin the version please :)

Comment (literat) [reply package.json:112]: Fixed.

**Comment by literat** [comment]
> I merged the previous PRs, could you please rebase?

Rebased.

**Review by OndraM — APPROVED** [review]

**Pull request merged**